cli/args/mod.rs

```diff
@@ -717,7 +717,11 @@ impl CliOptions {
 
     if let Some(env_file_name) = &flags.env_file {
       if (from_filename(env_file_name)).is_err() {
-        bail!("Unable to load '{env_file_name}' environment variable file")
+        println!(
```

Comment:
```suggestion
        log::info!(
```

It's better to output to stderr for messages. With println it could corrupt anyone piping the output of their program.

Reply:
Done